Sources/StreamVideoSwiftUI/Utils/PictureInPicture/StreamYUVToARGBConversion.swift

```diff
@@ -0,0 +1,128 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Accelerate
+import Foundation
+
+/// A class dedicated to converting YUV (YpCbCr) image data to ARGB format.
+final class StreamYUVToARGBConversion {
+
+    /// Enum defining the color conversion coefficients for YUV to ARGB conversion.
```

Comment:
Thanks for writing these explanations, helps a lot in understanding what you are doing 😅

Reply:
I tried my best to explain everything. Please let me know if there are parts that can be improved and i can look into them.